scripts/get_command_info.py

```diff
@@ -0,0 +1,210 @@
+# Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0
```

Comment:
Is this the correct license header? Or should it be different since this is redis-rs instead of glide-for-redis?

Reply:
I would omit the Copyright note